src/router.rs

```diff
@@ -66,4 +66,18 @@ impl<'a> Router<'a> {
 
         false
     }
+
+    /// Note that unlike `execute`, this doesn't set `Context.result`
+    /// but it also doesn't need a `&mut Context`.
+    pub fn r#match(&self, context: &Context) -> Option<Match> {
```

Comment:
Can we have another name rather than `match`? The raw identifier is too verbose to be used.

Reply:
Like `find` or `search`?

Reply:
Actually what about `try_match`?

Reply:
find/search sound too much like regex/globs and here we're executing "matchers"